ls8/cpu.c

```diff
@@ -9,22 +10,23 @@
  */
 void cpu_load(struct cpu *cpu)
 {
+/* printf(" argv => %s\n", argv[0] ); */
   char data[DATA_LEN] = {
     // From print8.ls8
     0b10000010, // LDI R0,8
     0b00000000, // 0
     0b00001000, //8
     0b01000111, // PRN R0
-    0b00000000, // 0
+    0b00000000, // 0k
     0b00000001  // HLT
   };
 
   int address = 0;
   for (int i = 0; i < DATA_LEN; i++) {
     cpu->ram[address++] = data[i];
-    /* printf(" cpu ram : %d \n", cpu->ram[address - 1]); */
+    printf(" cpu ram : %u \n", cpu->ram[address - 1]);
```

Comment:
nice. now, does this still run the program as expected?

Reply:
Yes. 👍